lib/browserify.js

```diff
@@ -6,12 +6,7 @@ const transform = require('./js-transform')
 
 module.exports = browserifyIt
 
-function browserifyIt ({
-  config,
-  entry,
-  env,
-  minify
-}) {
+function browserifyIt({config, entry, env, minify}) {
```

Comment:
The seats on United are so compact that sometimes they have to re-accomodate passengers.